btp/provider/resource_subaccount.go

```diff
@@ -370,6 +406,12 @@ func (rs *subaccountResource) Update(ctx context.Context, req resource.UpdateReq
 	}
 
 	plan, diags = subaccountValueFrom(ctx, updatedRes.(cis.SubaccountResponseObject))
+
+	if !state.SkipAutoEntitlement.IsUnknown() {
+		// The value was explicitly set in the state, so we keep it
+		plan.SkipAutoEntitlement = state.SkipAutoEntitlement
```

Comment:
```suggestion
		skipAutoEntitlement := plan.SkipAutoEntitlement.ValueBool()
		args.SkipAutoEntitlement = skipAutoEntitlement
```

Reply:
Args does not have the argument, will undo it